ConnectionManager.lib.nut

```diff
@@ -184,19 +184,34 @@ class ConnectionManager {
         return true;
     }
 
-    // Disconnects, and runs the onDisconnected handler
-    function disconnect() {
+    // Disconnects, and runs the onDisconnected handler.
+    // Does nothing if the imp is in process of connecting.
+    //
+    // Parameters:
+    //      force           Force disconnect regardless of whether the device
+    //                      already tries to connect now.
+    //      flushTimeout    The timeout used for `server.flush` call.
+    //
+    // Returns:         this
+    function disconnect(force = false, flushTimeout = CM_FLUSH_TIMEOUT) {
         // If we're connecting / disconnecting, try again in 0.5 seconds
```

Comment:
This code comment is confusing because it is not actually what happens if retryOnTimeout is set to false - in 0.5 seconds the watchdog runs.

Reply:
That's true. Just took it out. Thanks!